src/app_service/service/message/service.nim

```diff
@@ -679,7 +681,16 @@ proc renderInline(self: Service, parsedText: ParsedText): string =
 
   case parsedText.type:
     of "":
-      result = value
+      if value == MENTION_EVERYONE_LITERAL:
+        result = value.replace(MENTION_EVERYONE_LITERAL, MENTION_EVERYONE_TAG)
+      elif value.contains(fmt(" {MENTION_EVERYONE_LITERAL} ")):
+        result = value.replace(fmt(" {MENTION_EVERYONE_LITERAL} "), " " & MENTION_EVERYONE_TAG & " ")
+      elif value.startsWith(fmt("{MENTION_EVERYONE_LITERAL} ")):
+        result = value.replace(fmt("{MENTION_EVERYONE_LITERAL} "), MENTION_EVERYONE_TAG & " ")
+      elif value.endsWith(fmt(" {MENTION_EVERYONE_LITERAL}")):
+        result = value.replace(fmt(" {MENTION_EVERYONE_LITERAL}"), " " & MENTION_EVERYONE_TAG)
```

Comment:
There has to be a regex that does this. Something like
```
/ ?@everyone ?/
```

Reply:
For sure, I just suck at them! :D
Will improve this

Reply:
This is no longer needed after the latest changes to move parsing to of system mentions to the markdown parser